components/match_ticker/commons/match_ticker_display_components_new.lua

```diff
@@ -175,14 +177,31 @@ function Details:countdown()
 end
 
 ---@return Html?
-function Details:streams()
+function Details:streamsOrVods()
 	local match = self.match
-	local links = mw.html.create('div')
-		:addClass('match-streams')
 
-	links:wikitext(table.concat(StreamLinks.buildDisplays(StreamLinks.filterStreams(match.stream)) or {}))
+	if not Logic.readBool(match.finished) then
+		return mw.html.create('div')
+			:addClass('match-streams')
+			:wikitext(table.concat(StreamLinks.buildDisplays(StreamLinks.filterStreams(match.stream)) or {}))
+	end
+
+	local vods = mw.html.create('div')
+			:addClass('match-streams')
+
+	---@param obj table
+	---@param index integer?
+	local addVod = function(obj, index)
+		vods:node(Logic.isNotEmpty(obj.vod) and VodLink.display{
+			vod = obj.vod,
+			gamenum = index,
+		} or nil)
+	end
+
+	addVod(match)
+	Array.forEach(match.match2games or {}, addVod)
```

Comment:
Do we need to limit the amount of VODs shown? Space could get quite tight if there are more than 2 games with VODs

Reply:
fwiw if we have a match with several streams (i.e. on different platforms) the same applies too